app/screens/settings/about/about.tsx

```diff
@@ -171,6 +184,22 @@ const About = ({componentId, config, license}: AboutProps) => {
         popTopScreen(componentId);
     });
 
+    const copyToClipboard = useCallback(
+        () => {
+            const appVersion = `${intl.formatMessage({id: 'settings.about.version', defaultMessage: 'App Version:'})} ${intl.formatMessage({id: 'settings.about.build', defaultMessage: '{version} (Build {number})'},
+                {version: DeviceInfo.getVersion(), number: DeviceInfo.getBuildNumber()})}`;
+            const server = `${intl.formatMessage({id: 'settings.about.server.version.desc', defaultMessage: 'Server Version:'})} ${intl.formatMessage({id: serverVersion.id, defaultMessage: serverVersion.defaultMessage}, serverVersion.values)}`;
```

Comment:
```suggestion
            const buildNumber = config.BuildNumber;
            const version = config.Version;
            const server = buildNumber === version ?
                intl.formatMessage({id: 'settings.about.serverVersionNoBuild', defaultMessage: 'Server Version: {version}'}, {version}) :
                intl.formatMessage({id: 'settings.about.serverVersion', defaultMessage: 'Server Version: {version} (Build {buildNumber}'}, {version, buildNumber});
```

And remove the code that creates `serverVersion`around line 158.

Reply:
we need that, we are using that in rendering the message on screen, not only in this func